ada_2_fhir/zibs2017/payload/zib-plannedcareactivityfortransfer.xsl

```diff
@@ -0,0 +1,432 @@
+                                <xsl:call-template name="format2FHIRDate">
+                                    <xsl:with-param name="dateTime" select="xs:string(.)"/>
+                                    <xsl:with-param name="dateT" select="$dateT"/>
+                                    <xsl:with-param name="precision">seconds</xsl:with-param>
+                                </xsl:call-template>
+                            </xsl:attribute>
+                        </start>
+                    </xsl:for-each>
+                    <xsl:for-each select="$endDate">
+                        <end>
+                            <xsl:attribute name="value">
+                                <xsl:call-template name="format2FHIRDate">
+                                    <xsl:with-param name="dateTime" select="xs:string(.)"/>
+                                    <xsl:with-param name="dateT" select="$dateT"/>
+                                    <xsl:with-param name="precision">seconds</xsl:with-param>
+                                </xsl:call-template>
+                            </xsl:attribute>
+                        </end>
+                    </xsl:for-each>
+                    
+                    <xsl:for-each select="(reden_contact/afwijkende_uitslag | contact_reason/deviating_result)[@value]">
+                        <reason>
+                            <text>
+                                <xsl:call-template name="string-to-string">
+                                    <xsl:with-param name="in" select="."/>
+                                </xsl:call-template>
+                            </text>
+                        </reason>
+                    </xsl:for-each>
+                    
+                    <xsl:for-each select="reden_contact/probleem | contact_reason/problem">
+                        <xsl:choose>
+                            <xsl:when test="*">
+                                <indication>
+                                    <xsl:apply-templates select="." mode="doProblemReference-3.0"/>
+                                </indication>
+                            </xsl:when>
+                        </xsl:choose>
+                    </xsl:for-each>
+                    <xsl:for-each select="reden_contact/verrichting | contact_reason/procedure">
+                        <xsl:choose>
+                            <xsl:when test="*">
+                                <indication>
+                                    <xsl:apply-templates select="." mode="doProcedureReference-2.1"/>
+                                </indication>
+                            </xsl:when>
+                        </xsl:choose>
+                    </xsl:for-each>
+                    
+                    <participant>
+                        <actor>
+                            <xsl:apply-templates select="$adaPatient" mode="doPatientReference-2.1"/>
+                        </actor>
+                        <status value="accepted"/>
```

Comment:
In the profile this element has short description ContactWithStatus, but no mapping has been added and I cannot seem to find that element within the zibs. Is this an element from eAfspraak? For the BgZ however, _accepted_ seems perfectly fine.

Reply:
https://simplifier.net/packages/nictiz.fhir.nl.stu3.eafspraak/1.0.6/files/714361/~mappings - The mapping is in the [ART-DECOR Dataset eAfspraak](https://decor.nictiz.nl/art-decor/decor-datasets--eafspraken-?id=2.16.840.1.113883.2.4.3.11.60.61.1.1&effectiveDate=2017-08-01T00:00:00&language=nl-NL) to ContactStatus - which in itself inherits from the zib. Not ideal but it is there, somewhere